src/aeonlib/ocs/lco/instruments.py

```diff
@@ -1,17 +1,29 @@
 # pyright:  reportUnannotatedClassAttribute=false
 # This file is generated automatically and should not be edited by hand.
 
-from typing import Any, Annotated, Literal
+from typing import Any, Annotated, Literal, Optional
 
-from annotated_types import Le
-from pydantic import BaseModel, ConfigDict
+from annotated_types import Le, Ge
+from pydantic import BaseModel, ConfigDict, Field
 from pydantic.types import NonNegativeInt, PositiveInt
 
 from aeonlib.models import TARGET_TYPES
 from aeonlib.ocs.target_models import Constraints
 from aeonlib.ocs.config_models import Roi
 
 
+
+class Lco0M4ScicamQhy600ConfigExtraParams(BaseModel):
+    model_config = ConfigDict(validate_assignment=True, extra='allow')
+    sub_expose: Literal[[False, True]] = False
+    sub_exposure_time: Optional[Annotated[float, Ge(15.0)]] = None
```

Comment:
As per previous comment, this should be `sub_exposure_time: Annotated[float, Ge(15.0)] | None = None`